smoketest/scripts/cli/test_qos.py

```diff
@@ -279,6 +291,15 @@ def test_05_limiter(self):
                 if 'bandwidth' in qos_class_config:
                     self.cli_set(qos_class_base + ['bandwidth', qos_class_config['bandwidth']])
 
+                if 'exceed' in qos_class_config:
+                    self.cli_set(qos_class_base + ['exceed', qos_class_config['exceed']])
+
+                if 'burst' in qos_class_config:
+                    self.cli_set(qos_class_base + ['burst', qos_class_config['burst']])
+
+                if 'mtu' in qos_class_config:
+                    self.cli_set(qos_class_base + ['mtu', qos_class_config['mtu']])
```

Comment:
I see the MTU programmed to the kernel, but there is no validation who checks if the programmed value is the value when it's read back.

Reply:
done. Added validate what exactly is set to tc policy, not ideal but works.